### ISSUE TITLE
Bug: Custom Color not Applied

### ISSUE THREAD

**Issue by Devo7v** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#71]
When turning on the Custom Theme, the app is has a dark theme with green accents as the default.  If the user chooses another color (orange for instance) most of the accent colors change properly except for the check boxes in the settings and the word "Music"  in the "Google Play Music" slide out menu.  See the screenshot below.
Would it be possible to have the ability in the program to manually enter either a hex or rgb code for the custom color?
![screenshot 1](https://cloud.githubusercontent.com/assets/15945504/11545298/9b0037a2-9914-11e5-9aa4-74a77c30d1a6.jpg)

**Comment by MarshallOfSound** [comment]
The RGB idea was suggested somewhere but can't seem to find it in an open issue so I'll leave this one open to track it.  

As for the "Music" in Google Play Music.  Theming that part it virtually impossible because it is by default orange and the only way I can change it is using CSS filters to adjust hue/saturation/brightness.  I'm looking into replicating the image as an SVG to make colours easier.

The checkboxes aren't actually part of the theme, I can make them part of it but green is a global signifier for good / active.  I think it is just confusing that the default highlight is green as well (The two greens are actually completely different)

TLDR: Hex color = Will do
           Music = Working on
           Checkboxes = Invalid
